experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -85,30 +86,38 @@ if mode_name == "amip"
     println("No ocean sim - do not expect energy conservation")
 
     # ocean
-    weightfile, datafile_cgll, regrid_space =
-        ncreader_rll_to_cgll_from_space(sst_data, "SST", boundary_space, outfile = "sst_cgll.nc")
-    SST_init = ncreader_cgll_sparse_to_field(datafile_cgll, "SST", weightfile, (Int(1),), regrid_space)[1]
-    SST_init = swap_space!(SST_init, axes(mask)) .* (abs.(mask .- 1)) .+ FT(273.15)
-
+    SST_info = bcfile_info_init(
+        sst_data,
+        "SST",
+        boundary_space,
+        segment_idx0 = [Int(1309)],
+        interpolate_monthly = false,
+        scaling_function = clean_sst,
+    )
+    update_midmonth_data!(date0, SST_info)
+    SST_init = deepcopy(SST_info.monthly_fields[1])
     ocean_params = OceanSlabParameters(FT(20), FT(1500.0), FT(800.0), FT(280.0), FT(1e-3), FT(1e-5), FT(0.06))
-    ocean_sim = (; integrator = (; u = (; T_sfc = SST_init), p = (; params = ocean_params)))
-    mode_specifics = (; name = mode_name)
+    ocean_sim = (; integrator = (; u = (; T_sfc = SST_init), p = (; params = ocean_params), SST_info = SST_info))
 
     # sea ice
-    # (Currently, we only support a slab ice model with fixed area and depth.)
-    weightfile, datafile_cgll, regrid_space =
-        ncreader_rll_to_cgll_from_space(sic_data, "SEAICE", boundary_space, outfile = "sic_cgll.nc")
-    SIC = ncreader_cgll_sparse_to_field(datafile_cgll, "SEAICE", weightfile, (Int(1),), regrid_space)[1]
-    SIC = swap_space!(SIC, axes(mask)) .* (abs.(mask .- 1))
-    ice_mask = get_ice_mask.(SIC .- FT(25), FT) # here 25% and lower is considered ice free
-
+    SIC_info = bcfile_info_init(
+        sic_data,
+        "SEAICE",
+        boundary_space,
+        segment_idx0 = [Int(1309)],
+        interpolate_monthly = false,
+        scaling_function = clean_sic,
+    )
+    update_midmonth_data!(date0, SIC_info)
+    SIC_init = deepcopy(SIC_info.monthly_fields[1])
+    ice_mask = get_ice_mask.(SIC_init .- FT(50), FT) # here 50% and lower is considered ice free
     ice_sim = ice_init(FT; tspan = tspan, dt = Δt_cpl, space = boundary_space, saveat = saveat, ice_mask = ice_mask)
+    mode_specifics = (; name = mode_name, SST_info = SST_info, SIC_info = SIC_info)
 
-else
-    # ocean
+elseif mode_name == "aquaplanet"
```

Comment:
is "aquaplanet" a misleading name at all? Not sure the convention. but this "aquaplanet" has land and ice as well.

Reply:
I actually removed the ice. Since one of the goals was to have a full aquaplanet, shall we remove the land too? It means land won't be included in the conservation check though. Maybe  going forward, we can have another case like `simple_earth`, which demonstrates the conservation of all slab+Bucket components, also with topography?

Reply:
yes! I think both are great ideas. We can have a full aquaplanet and a full land planet, and then AMIP?